### PR TITLE
Fix an indentation problem

### PR DESCRIPTION
**Pull request by XuanlinGuan**
Fix an indentation problem

### CODE REVIEW

**Comment by beinan** [comment]
@XuanlinGuan could you sign the CLA? I will review your pr very soon, thx!

File: core/common/src/main/java/alluxio/worker/block/BlockStore.java [hunk 1]
```diff
@@ -111,7 +111,8 @@ BlockReader createBlockReader(long sessionId, long blockId, long offset,
    * @throws IOException if it fails to get block reader
    */
   BlockReader createUfsBlockReader(long sessionId, long blockId, long offset, boolean positionShort,
-                                   Protocol.OpenUfsBlockOptions options) throws IOException;
+                                   Protocol.OpenUfsBlockOptions options)
+      throws IOException;
```
Comment (beinan) [inline core/common/src/main/java/alluxio/worker/block/BlockStore.java:115]: The indentation is still not fully correct
```suggestion
      Protocol.OpenUfsBlockOptions options) throws IOException;
```

**Review by beinan — CHANGES_REQUESTED** [review]

**Review by beinan — APPROVED** [review]
lgtm, thx!

**Comment by beinan** [comment]
alluxio-bot, merge this please.

**Comment by Xenorith** [comment]
signed cla has been received @Zijian-Zhu @HelloHorizon fya

**Comment by beinan** [comment]
alluxio-bot, merge this please.

**Pull request merged**